### PR TITLE
Fix for latest Bakkesmod update

### PR DESCRIPTION
**Pull request by stevenschemers**

### CODE REVIEW

**Comment by yonilerner** [comment]
Thank you for this! I will review and upload shortly

**Comment by stevenschemers** [comment]
> Thank you for this! I will review and upload shortly

No problem at all, thank you for the plugin!

File: bakkesmod-playlist-rank-viewer/bakkesmod-playlist-rank-viewer.vcxproj [hunk 1]
```diff
@@ -23,33 +23,33 @@
...
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
     <ProjectName>bakkesmod-playlist-rank-viewer</ProjectName>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v141</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```
Comment (yonilerner) [inline bakkesmod-playlist-rank-viewer/bakkesmod-playlist-rank-viewer.vcxproj:33]: I actually use VS2017 so this breaks my builds :) Im going to try and install VS2019 and hope that fixes it though. Will get back to you

**Comment by yonilerner** [comment]
Only other comment is that I want to keep the built .dll in there. I know its generally bad form to commit binaries, but given the distribution method for this plugin is really slow to update since it requires a manual code review, I like just committing the dll so anyone can download newer versions immediately if they want

**Comment by yonilerner** [comment]
Scratch the last comment, I misread the diff :/ This looks good. Builds properly and seems to still work. Thank you!

**Pull request merged**